middleware/profiler.go

```diff
@@ -11,13 +10,13 @@ import (
 
 // Profiler is a convenient subrouter used for mounting net/http/pprof. ie.
 //
-//  func MyService() http.Handler {
```

Comment:
It was fixed by VS Code automatically from spaces to tabs. If it is important to be as before, I can revert this comment back via `vim`